test/unit/error-tests.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var assert = require('assert');
+var path = require('path');
+var errors = require('../../lib/errors');
+var helper = require('../test-helper');
+var fileName = path.basename(__filename);
+
+describe('DriverError', function () {
+  it('should inherit from Error and have properties defined', function () {
+    var error = new errors.DriverError('My message');
+    assertError(error, errors.DriverError);
+  });
+});
+describe('NoHostAvailableError', function () {
+  it('should inherit from DriverError and have properties defined', function () {
```

Comment:
To effectively ensure this inherits from `DriverError` shouldn't `assertError` also check for that?    I see `helper.assertInstanceOf(err, Error);` but not `helper.assertInstanceOf(err, errors.DriverError);`

Reply:
My intention was to assert instance of `DriverError`, I've pushed a fix.